modular_sand/code/datums/interactions/lewd_definitions.dm

```diff
@@ -34,6 +34,7 @@
 	var/lastlusttime = 0
 	var/lust = 0
 	var/multiorgasms = 1
+	var/hair_trigger_mul = 1
```

Comment:
Rather than adding a variable to the mob you should probably just check if the mob has the trait when the add_lust proc is called and multiply the added lust if so